### DISCUSSION TITLE
[Bug] upvoting discussions doesn't reflect on a discussion

### DISCUSSION THREAD

**Discussion by Milo123459** [community/community#21116] [Discussions]
https://user-images.githubusercontent.com/50248166/180598151-eb64bb89-f7c2-4d66-8acf-a37b67fddb3e.mp4

**Comment by AyushSehrawat** [comment]
Bug on your end probably. Works fine for me. Try removing cache and log in again

**Comment by MrSarno** [comment]
Can confirm that I'm also unable to reproduce this.

Could be a cache issue, as suggested by @AyushSehrawat. Could be a browser setting or plugin interfering.
